tests/functional_tests/cpp_backend_tests/CMakeLists.txt

```diff
@@ -26,12 +26,12 @@ FetchContent_MakeAvailable(GridTools)
 
 function(generate_computation)
     set(options)
-    set(oneValueArgs NAME SRC_FILE GENERATED_FILENAME)
+    set(oneValueArgs NAME SRC_FILE GENERATED_FILENAME IMPERATIVE)
```

Comment:
This should go to `options` as it's a boolean flag.

Reply:
Maybe I'm doing things wrong, but if I add `IMPERATIVE` to args it gets a default value of False, and I'd prefer not to adapt the Cartesian tests, too. The point is that this is an _optional_ boolean Flag.

Reply:
Not, sure I understand, but I guess forwarding OPTIONS is more cumbersome, so maybe let's keep it.